routes/restaurant/router.go

```diff
@@ -71,5 +71,10 @@ func (r restaurantRouter) createRestaurant(w http.ResponseWriter, req *http.Requ
 		Logo:   &file,
 	})
 
-	httputils.JSON(w, id)
+	err = httputils.JSON(w, id)
+	
+	if err != nil {
+        http.Error(w, "Internal Server Error", http.StatusInternalServerError)
+        return
```

Comment:
not sure if this will be the best handling for this error in case returning http response failed.

Reply:
was thinking to retry then maybe return this maybe we can check how to add retry policy.

Reply:
Make sense